packages/esm-patient-chart-app/src/ui-components/patient-chart/patient-chart.component.scss

```diff
@@ -35,10 +35,6 @@
   flex-direction: column;
 }
 
-:global(.omrs-breakpoint-gt-tablet) .chartContainer {
-  margin-left: var(--omrs-sidenav-width);
-}
-
```

Comment:
@FlorianRappl , this is the global prop I removed that gave the left margin to the patient-chart page.